.github/workflows/android-cd.yml

```
@@ -41,4 +41,4 @@ jobs:
                   serviceCredentialsFileContent: ${{ secrets.CREDENTIAL_FILE_CONTENT }}
                   groups: "testersteam"
                   file: app/build/outputs/apk/debug/app-debug.apk
-                  releaseNotes: $TITLE
+                  releaseNotes: ${{ TITLE }}
```

actionlint (1.7.9)

undefined variable "TITLE". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)